### PR TITLE
WEB-625: Rebrand and rewrite Onboarding API Guide

### PR DESCRIPTION
**Pull request by aurorawalker**

### CODE REVIEW

**Comment by aurorawalker** [comment]
@brianorwhatever  if you could do a final review of this documentation, specifically looking for correctness, this pull request is ready to be merged into the careless-whisper-rebrand branch.

You can view these changes here: http://dev.beanstream.com.cwguide.s3-website-us-east-1.amazonaws.com/docs/guides/onboarding/

File: source/docs/guides/onboarding.md [hunk 1]
```diff
@@ -13,124 +14,271 @@ navigation:
 ---
 
 # Onboarding
```
Comment (brianorwhatever) [inline source/docs/guides/onboarding.md]: Should this say Onboarding API?

File: source/docs/guides/onboarding.md [hunk 2]
```diff
@@ -13,124 +14,271 @@ navigation:
...
+* The integration submits complete and correct applications.
+* The integration displays or links to the Terms and Conditions.
+* The integration displays or links to the prices for card processing.
 
+Once an integration is approved, you will be emailed an API key for the Production environment.
+
+**Step 4:** Update to Production.
+   Update your integration for Production using your Production API key. A test application should be completed on the
+   Production environment. Please produce a sample test application and then send to your Account Manager the application
+   id and the flow used in the test. When the test is complete, we will remove the test data.
```
Comment (brianorwhatever) [inline source/docs/guides/onboarding.md]: Are we sure we want this? Who will be deleting the data? I don't know if this test is necessary if they are successfully playing in sandbox

File: source/docs/guides/onboarding.md [hunk 3]
```diff
@@ -13,124 +14,271 @@ navigation:
...
+* **PSP-CAD** for Payment Service Providers in Canada
+* **PSP-USD** for Payment Service Providers in the United States
 
-## Required fields
-Most, but not all, fields listed in the API spec are required. You do not need to submit all required fields with each request, but in order to get an application to the in_review state, all required fields must be populated.
+### Gateway
 
-## Updating and deleting fields
-When you update an application all fields that you pass in will be saved. If there are fields that you do not want updated, then simply do not pass them. If there are fields that you previously passed, but now want emptied, then set their value to null or zero. Here is an example JSON request removing some fields:
+Gateway flows are similar to card payments but the merchant account is held with another acquiror. Accounts created with these
```
Comment (brianorwhatever) [inline source/docs/guides/onboarding.md]: Is acquiror the correct spelling? Googling it confused me more but I always thought it was acquirer

File: source/docs/guides/onboarding.md [hunk 4]
```diff
@@ -13,124 +14,271 @@ navigation:
...
+## Sample Integration Calls
+
+Let’s look at a flow of API for an application for a merchant account provisioned to accept card payments in 
+Canadian dollars (PSP-CAD). 
+
+### 1. Terms and Conditions
+**GET** `/v1/boarding/workflows/psp-cad/terms/{partner}/{file_type}/version`
+
+You need to either display or link to the Terms and Conditions in your user interface. The API requires that you specify 
+the exact version of the Terms And Conditions that your sub-merchant has agreed to, as well as the date and time at
```
Comment (brianorwhatever) [inline source/docs/guides/onboarding.md]: capital on And doesn't match other instances

File: source/docs/guides/onboarding.md [hunk 5]
```diff
@@ -139,69 +287,153 @@ curl https://uat-onboarding-api.beanstream.com/v1/workflows/psp-cad/applications
...
+
+You can execute the following cURL command by filling in your version, authorization header, and application id:
+
+```curl
+curl https://sandbox-api.na.bambora.com/v1/boarding/workflows/psp-cad/applications/{applicationId} \
+-X PUT \
+-H "Authorization: Bearer YOUR_TOKEN_HERE" \
+-H "Content-Type: application/json" \
+-H "X-API-Version : SELECTED_VERSION_HERE" \
+-d '{"applicant": {
```
Comment (brianorwhatever) [inline source/docs/guides/onboarding.md]: Here and below the curl requests start the data with the first key on the same line as the bracket whereas previously they were broken out

File: source/docs/guides/onboarding.md (hunk 1 above)
Comment (aurorawalker) [inline source/docs/guides/onboarding.md]: There already is an 'Onboarding API' which links to the actual API spec. But maybe I can still do better with this name. The others follow the pattern 'Quickstart - Service', so I could make this 'Quickstart -Onboarding'

File: source/docs/guides/onboarding.md (hunk 2 above)
Comment (aurorawalker) [inline source/docs/guides/onboarding.md]: I am not sure we want this. This section was taken from the previous version of this documentation. You make a good point about the sandbox - I think that's argument enough to remove this line.

File: source/docs/guides/onboarding.md (hunk 3 above)
Comment (aurorawalker) [inline source/docs/guides/onboarding.md]: haha - yes. I actually spoke to three people about this. Acquiror is the UK/Canadian spelling and acquirer is US spelling. It was confirmed that we use Canadian spelling for our documentation.

File: source/docs/guides/onboarding.md (hunk 4 above)
Comment (aurorawalker) [inline source/docs/guides/onboarding.md]: good catch, fixed

File: source/docs/guides/onboarding.md (hunk 5 above)
Comment (aurorawalker) [inline source/docs/guides/onboarding.md]: fixed

**Pull request merged**